### ISSUE TITLE
Alias to qualified module does not work

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#3669]
This program generates an unexpected error:

```py
import os.path
x = os.path
x.isdir('/') # Module has no attribute "isdir"
```

However, this works:

```py
from os import path
x = path
x.isdir('/') # Ok
```

Module aliasing was implemented in #3435.

**Comment by ilevkivskyi** [comment]
I think this might be an one-line fix. Just use ``RefExpr`` instead of ``NameExpr`` in ``semanal.process_module_assignment``. I can take a look at it this evening.

**Comment by ilevkivskyi** [comment]
OK, it is _slightly_ more complex than I thought, here is the PR #3680.
